handlebars/pom.xml

```diff
@@ -89,6 +103,7 @@
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
+      <scope>provided</scope>
```

Comment:
Would you be amendable to a separate pom that produced an artifact that was just for OSGI? Otherwise I don't think we'd really be able to sort out all the things that could come from OSGI vs. what is required for normal environments.